webapp/src/Controller/API/BalloonController.php

```diff
@@ -28,6 +28,7 @@ class BalloonController extends AbstractRestController
     /**
      * Get all the balloons for this contest.
      *
+     * @return array<int, int|string>
```

Comment:
This does not return an array of strings? I think it returns an array of arrays. But maybe again we should use a DTO.